src/UnscentedKalmanInversion.jl

```diff
@@ -172,7 +172,16 @@ function construct_sigma_ensemble(
 
     c_weights = process.c_weights
 
-    chol_xx_cov = cholesky(Hermitian(x_cov)).L
+    # compute cholesky factor L of x_cov
+    try
+        global chol_xx_cov = cholesky(Hermitian(x_cov)).L
```

Comment:
Is it safe to use `global` here @charleskawczynski ?

Reply:
I will add a unit test, if I do not put global here, I get an error. the scope of try and catch is weird

Reply:
I would typically use `local` here, maybe global works the same way? I think `local` is more intuitive.

Reply:
just update to `local` and add a test